### PR TITLE
Ensure GA titles are set for other views

### PR DESCRIPTION
**Pull request by miltontony**

### CODE REVIEW

**Comment by miltontony** [comment]
@Rizziepit review please

File: springboard/tests/test_views.py [hunk 1]
```diff
@@ -96,3 +98,41 @@ def test_multiple_repos(self):
...
+        [category] = self.mk_categories(self.workspace, count=1, position=1)
+        [page] = self.mk_pages(
+            self.workspace, count=1, position=1,
+            created_at=datetime.utcnow().isoformat(),
+            primary_category=category.uuid)
+
+        app = self.mk_app(
+            self.workspace,
+            settings={'ga.profile_id': 'ID-000',
+                      'ga.persona_dimension_id': 'dimension0'})
```
Comment (Rizziepit) [inline springboard/tests/test_views.py]: That `ga.persona_dimension_id` is specific to IoGT. Might be confusing to have it in tests here.

**Comment by Rizziepit** [comment]
+1

File: springboard/tests/test_views.py (hunk 1 above)
Comment (miltontony) [inline springboard/tests/test_views.py]: copy pastey!

**Pull request merged**